book/guided-tour/README.md

```diff
@@ -37,12 +37,13 @@ but it's worth understanding the differences between these libraries.
   same time, it has many more dependencies, and so takes longer to
   build, and will add more to the size of your executables.
 
-As of this writing, the stable release of `Core` is less portable than
-`Base`, running only on UNIX-like systems.  For that reason, there's
-yet a third library, `Core_kernel`, which is a portable subset of
-`Core`. That said, the latest development version removes
-`Core_kernel`, and makes `Core` itself portable.  The stable release
-should have `Core_kernel` removed by mid-2022, so we won't use
+As of the version of `Base` and `Core` used in this book (version
+`v0.14.0`), `Core` is less portable than `Base`, running only on
```

Comment:
Base is actually v0.14.3 -- perhaps just drop the `.0` suffix here.

Reply:
Are you sure? That's not what base.opam says in the duniverse:

```
opam-version: "2.0"
version: "v0.14.0"
maintainer: "opensource@janestreet.com"
authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
homepage: "https://github.com/janestreet/base"
bug-reports: "https://github.com/janestreet/base/issues"
dev-repo: "git+https://github.com/janestreet/base.git"
doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/base/index.html"
license: "MIT"
```